FreeRTOS/FreeRTOS
# coreMQTT_Windows_Simulator Update

coreMQTT_Windows_Simulator Update

Description
-----------
Changes to the solution to import in static versions of libraries + kernel

Test Steps
-----------
Open demo, set necessary variables, run the demo.


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
